### ISSUE TITLE
php-fpm fails to start due to missing directory /run/php

### ISSUE THREAD

**Issue by chrisinmtown** [coolacid/docker-misp#117]
I am trying to run the MISP docker container tagged 2.4.143 on a mac osx environment with docker version 19.03.14 and docker-machine version 0.16.2. The container script `entrypoint_fpm.sh` runs command `/usr/sbin/php-fpm7.3 -R -F` which fails:
```
misp_1   | [24-May-2021 12:36:46] ERROR: unable to bind listening socket for address '/run/php/php7.3-fpm.sock': No such file or directory (2)
misp_1   | [24-May-2021 12:36:46] ERROR: FPM initialization failed
misp_1   | 2021-05-24 12:36:46,599 INFO exited: php-fpm_00 (exit status 78; not expected)
```
I exec'd into that running container and found that /run/php did not exist.  I don't know why the php-fpm stuff doesn't create it. My workaround was to add the following lines to the top of that script:
```
fpmdir="/run/php"
if [ ! -d $fpmdir ] ; then
    echo "create $fpmdir"
    mkdir $fpmdir
fi
```
I would like to ask, if php-fpm fails to start like this, can `supervisor` recognize that the whole process failed and stop there?

**Comment by coolacid** [comment]
I'm not that far up on supervisor, I'm sure there's some configurations to make sure directories exist. I'll look into it.

**Comment by chrisinmtown** [comment]
Never mind my question about supervisor, and the environment (mac blah blah) probably is irrelevant. I should have posted what "FROM" base image I was using, this is now weeks ago and I don't remember.

tl;dr - the /run/php directory must be present in the image. If the little code snippet helps then please do use it, otherwise just close this.

**Comment by coolacid** [comment]
Cool, thanks.
